src/test/tests/maven.test.ts

```diff
@@ -360,4 +353,61 @@ describe('Maven Tests', async () => {
             ]
         ];
     }
+
+    describe('Update to fixed version', async () => {
+        const expectedVersion: string = '3.0.16';
+
+        it('Without version property', async () => {
+            const [mavenDependencyUpdate, issueToUpdate, pomXmlPath] = await setupTestEnvironment('updateToFixVersion');
+
+            // Operate the test
+            mavenDependencyUpdate.update(issueToUpdate, expectedVersion);
+
+            // Check results
+            const fileContent: string = fs.readFileSync(pomXmlPath, 'utf-8');
+            assert.include(fileContent, `<version>${expectedVersion}</version>`);
+            assert.isFalse(fs.existsSync(path.join(__dirname, '..', 'resources', 'maven', 'updateToFixVersion', 'pom.xml.versionsBackup')));
+        });
+
+        it('With property', async () => {
+            const [mavenDependencyUpdate, issueToUpdate, pomXmlPath] = await setupTestEnvironment('updateToFixVersionWithProperty');
+
+            // Operate the test
+            mavenDependencyUpdate.update(issueToUpdate, expectedVersion);
+
+            // Check results
+            const fileContent: string = fs.readFileSync(pomXmlPath, 'utf-8');
+            assert.include(fileContent, `<lets.go.and.fix.this>${expectedVersion}</lets.go.and.fix.this>`);
+            assert.isFalse(
+                fs.existsSync(path.join(__dirname, '..', 'resources', 'maven', 'updateToFixVersionWithProperty', 'pom.xml.versionsBackup'))
+            );
+        });
```

Comment:
Let's add tests for multi-module. I think most of the Maven projects are multi-modules.